### PR TITLE
Feature/extract components from cv pages

### PR DESCRIPTION
**Pull request by MontJamesmont**

### CODE REVIEW

File: src/cv/educationContainer/EducationContainer.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Education from "./education/Education";
+import './EducationContainer.css';
+import StandardSection from "../standardSection/StandardSection";
+
+function EducationContainer(props) {
+    const userEducations = props.userEducations;
+    const educationSource = props.educationSource;
```
Comment (StudentDawid) [inline src/cv/educationContainer/EducationContainer.js]: Better way:
const {userEducations, educationSource} = props;

File: src/cv/educationContainer/EducationContainer.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    let educationList = [];
+
+    if (userEducations) {
+        educationList = userEducations.map(function (education, index) {
```
Comment (StudentDawid) [inline src/cv/educationContainer/EducationContainer.js]: Better way :
const educationList = user. education ? user.education.map(function (education, index) {...}) : [];

File: src/cv/educationContainer/education/Education.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import './Education.css';
+
+import circleGreyMiddle from '../../../img/circle-grey-middle.svg';
+
+class Education extends Component {
+    render() {
+        let { education } = this.props;
```
Comment (StudentDawid) [inline src/cv/educationContainer/education/Education.js]: Can be const

File: src/cv/expectationContainer/ExpectationContainer.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './ExpectationContainer.css';
+import StandardSection from "../standardSection/StandardSection";
+
+function ExpectationContainer(props) {
+    const sourcesTarget = props.sourcesTarget;
+    const professionalExpectations = props.professionalExpectations;
```
Comment (StudentDawid) [inline src/cv/expectationContainer/ExpectationContainer.js]: Better way:
const {sourcesTarget, professionalExpectations} = props;

File: src/cv/experienceContainer/ExperienceContainer.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Experience from "./experience/Experience";
+import './ExperienceContainer.css';
+import StandardSection from "../standardSection/StandardSection";
+
+function ExperienceContainer(props) {
+    const userExperience = props.userExperience;
+    const experienceSource = props.experienceSource;
```
Comment (StudentDawid) [inline src/cv/experienceContainer/ExperienceContainer.js]: Better way:
const {userExperience, experienceSource} = props;

File: src/cv/experienceContainer/ExperienceContainer.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    let experiencesList = [];
+
+    if (userExperience) {
+        experiencesList = userExperience.map(function (experience, index) {
```
Comment (StudentDawid) [inline src/cv/experienceContainer/ExperienceContainer.js]: Better way :
const experiencesList = user.experience ? user.experience.map(function (experience, index) {...}) : [];

File: src/cv/experienceContainer/experience/Experience.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import './Experience.css';
+
+import circleGreyMiddle from '../../../img/circle-grey-middle.svg';
+import circleMaroonMiddle from '../../../img/circle-maroon-middle.svg';
+
+class Experience extends Component {
+    render() {
+        let { experience } = this.props;
```
Comment (StudentDawid) [inline src/cv/experienceContainer/experience/Experience.js]: Can be const

File: src/cv/personalNoteContainer/PersonalNoteContainer.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './PersonalNoteContainer.css';
+import StandardSection from "../standardSection/StandardSection";
+
+function PersonalNoteContainer(props) {
+    const sourcesProfile = props.sourcesProfile;
+    const userPersonalNote = props.userPersonalNote;
```
Comment (StudentDawid) [inline src/cv/personalNoteContainer/PersonalNoteContainer.js]: Better way:
const {sourcesProfile, userPersonalNote} = props;

File: src/cv/skillsetContainer/SkillsetContainer.js [hunk 9]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Skillset from "./skillset/Skillset";
+import './SkillsetContainer.css';
+
+function SkillsetContainer(props) {
+    const sourcesSkillset = props.sourcesSkillset;
+    const userSkillset = props.userSkillset;
```
Comment (StudentDawid) [inline src/cv/skillsetContainer/SkillsetContainer.js]: Better way:
const {sourcesSkillset, userSkillset} = props;

File: src/cv/skillsetContainer/SkillsetContainer.js [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+    let skillsetList = [];
+
+    if (userSkillset) {
+        skillsetList = (
```
Comment (StudentDawid) [inline src/cv/skillsetContainer/SkillsetContainer.js]: Better way :
const skillsetList = userSkillset ? (<Skillset skillset={userSkillset} />) : [];

File: src/cv/skillsetContainer/skillset/Skillset.js [hunk 11]
```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import './Skillset.css';
+
+import circleGreyMiddle from '../../../img/circle-grey-middle.svg';
+import circleMaroonMiddle from '../../../img/circle-maroon-middle.svg';
+
+class Skillset extends Component {
+    render() {
+        let { skillset } = this.props;
```
Comment (StudentDawid) [inline src/cv/skillsetContainer/skillset/Skillset.js]: Can be const

File: src/cv/standardSection/StandardSection.js [hunk 12]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './StandardSection.css';
+
+function StandardSection(props) {
+    const title = props.title;
+    const imgSrc = props.imgSrc;
+    const content = props.content;
```
Comment (StudentDawid) [inline src/cv/standardSection/StandardSection.js]: Better way:
const {title, imgSrc, content} = props;

File: src/cv/titleContainer/TitleContainer.js [hunk 13]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './TitleContainer.css';
+
+function TitleContainer(props) {
+    const sources = props.sources;
+    const user = props.user;
```
Comment (StudentDawid) [inline src/cv/titleContainer/TitleContainer.js]: Better way:
const {sources, user} = props;

**Review by StudentDawid — CHANGES_REQUESTED** [review]

File: src/cv/educationContainer/EducationContainer.js (hunk 1 above)
Comment (MontJamesmont) [reply src/cv/educationContainer/EducationContainer.js]: fixed in 7705c5ea5848f4633cca70b667caf424c1187e85

File: src/cv/educationContainer/EducationContainer.js (hunk 2 above)
Comment (MontJamesmont) [reply src/cv/educationContainer/EducationContainer.js]: fixed in 7705c5e

File: src/cv/educationContainer/education/Education.js (hunk 3 above)
Comment (MontJamesmont) [reply src/cv/educationContainer/education/Education.js]: fixed in 7705c5e

File: src/cv/expectationContainer/ExpectationContainer.js (hunk 4 above)
Comment (MontJamesmont) [reply src/cv/expectationContainer/ExpectationContainer.js]: fixed in 7705c5e

File: src/cv/experienceContainer/ExperienceContainer.js (hunk 5 above)
Comment (MontJamesmont) [reply src/cv/experienceContainer/ExperienceContainer.js]: fixed in 7705c5e

File: src/cv/skillsetContainer/SkillsetContainer.js (hunk 9 above)
Comment (MontJamesmont) [reply src/cv/skillsetContainer/SkillsetContainer.js]: fixed in 7705c5e

File: src/cv/skillsetContainer/SkillsetContainer.js (hunk 10 above)
Comment (MontJamesmont) [reply src/cv/skillsetContainer/SkillsetContainer.js]: fixed in 7705c5e

File: src/cv/skillsetContainer/skillset/Skillset.js (hunk 11 above)
Comment (MontJamesmont) [reply src/cv/skillsetContainer/skillset/Skillset.js]: fixed in a3289ca5011e616ec4192b2eece768266c6cf690

File: src/cv/standardSection/StandardSection.js (hunk 12 above)
Comment (MontJamesmont) [reply src/cv/standardSection/StandardSection.js]: fixed in 7705c5e

File: src/cv/titleContainer/TitleContainer.js (hunk 13 above)
Comment (MontJamesmont) [reply src/cv/titleContainer/TitleContainer.js]: fixed in 7705c5e

**Review by StudentDawid — APPROVED** [review]

**Pull request merged**